docs/stylesheets/dark-theme.scss

```diff
@@ -1,9 +1,59 @@
-$link_color: #FF7A04;
-$header_background: #FF7A04;
+$accent: #FF7A04;
+
+$link_color: $accent;
+$header_background: $accent;
 $background_color1: #2B2B2B;
 $background_color2: #1E2022;
-$navbar_header_background: $link_color;
+$navbar_header_background: $accent;
+
+/* Extraced fonts from Google */
+/* roboto-regular - cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Roboto'), local('Roboto-Regular'),
+       url('./fonts/roboto-v20-cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
+       url('./fonts/roboto-v20-cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin-regular.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+
+/* roboto-italic - cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin */
+@font-face {
+  font-family: 'Roboto';
+  font-style: italic;
+  font-weight: 400;
+  src: local('Roboto Italic'), local('Roboto-Italic'),
+       url('./fonts/roboto-v20-cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin-italic.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
+       url('./fonts/roboto-v20-cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin-italic.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+
+/* roboto-700 - cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Roboto Bold'), local('Roboto-Bold'),
+       url('./fonts/roboto-v20-cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin-700.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
+       url('./fonts/roboto-v20-cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin-700.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+
+/* roboto-mono-regular - cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin */
+@font-face {
+    font-family: 'Roboto Mono';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Roboto Mono'), local('RobotoMono-Regular'),
+         url('./fonts/roboto-mono-v7-cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
+         url('./fonts/roboto-mono-v7-cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin-regular.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
 
+body, input {
+    font-family: "Roboto", "Helvetica Neue", Helvetica, Arial, sans-serif
+}
+
+code, kbd, pre {
+    font-family: "Roboto Mono", "Courier New", Courier, monospace
+}
```

Comment:
I suggest moving all that font related stuff to a separate file. Just create a new `fonts.css` inside the `stylesheets` dir.  
That way this file only contains color theme related content.

You'll then need to add that stylesheet to the [`extra_css` section inside the mkdocs.yml](https://github.com/codeoverflow-org/chatoverflow-wiki/blob/f019b7e900b70925e13966d67130623a0f274c77/mkdocs.yml#L36-L38).